.github/workflows/e2e.yaml

```diff
@@ -30,6 +30,9 @@ jobs:
                     - 'e2e/parallel-custom-config'
                     - 'e2e/parallel-reflection-resolver'
                     - 'e2e/plain-views'
+                    - 'e2e/psr-4'
+                    - 'e2e/template-extends'
+                    - 'e2e/use-rector-configurator'
```

Comment:
`e2e/template-extends` and `e2e/use-rector-configurator` no longer exists, can be removed